src/store/index.js

```diff
@@ -1,4 +1,44 @@
 import { createStore } from 'vuex'
+import app from '@/helpers/axiosConfig'
+
+const pokemons = {
+  namespaced: true,
+  state: {
+    list: [],
+    pokemon: null,
+    abilities: [],
+  },
+  mutations: {
+    addPokemon(state, pokemon) {
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
+      if(pokemons){
+        state.list = pokemons
+      }
+      state.list.unshift(pokemon)
+      localStorage.setItem('pokemons', JSON.stringify(state.list))
+    },
+    getPokemon(state, name) {
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
+      if(pokemons){
+        state.pokemon = pokemons.find(pokemon => pokemon.name === name)
+      } else state.pokemon = state.list.find(pokemon => pokemon.name === name)
+      state.abilities = [
+        {title:'abilities', abilities: state.pokemon.abilities, key: 'ability'}, 
```

Comment:
An abilities array containing abilities, moves and stats, maybe it shouldn't be called abilities?

Reply:
that´s right.